packages/react/src/components/ListItem/index.js

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import PropTypes from "prop-types";
+import cx from "classnames";
+import { Icon } from "..";
+import { ICONS } from "../../constants/";
+
+const ListItem = ({
+  active,
+  title,
+  subtitle,
+  subitem,
+  miniitem,
+  value,
+  leftIcon,
+  rightIcon,
+  hideDivider,
+  onClick,
+}) => (
+  <div
+    onClick={onClick}
+    className={cx({
+      ListItem: true,
+      "ListItem--subitem": subitem && !miniitem,
+      "ListItem--miniitem": miniitem,
+      "is-active": active,
+      "is-clickable": !!onClick,
+      "ListItem-divider": !hideDivider,
+    })}
+  >
+    {leftIcon && (
+      <div className="ListItem-leftIcon">
+        <Icon icon={leftIcon} />
+      </div>
+    )}
+    <div>
+      <div className="ListItem-title">{title}</div>
+      {!!subtitle && <div className="ListItem-subtitle">{subtitle}</div>}
+    </div>
+    <div className="ListItem-valueContainer">
```

Comment:
What about moving this into a `rightEnhancer` prop?
So the user can have a larger control on what he wants to display?

Reply:
I was thinking about it, but in the end I thought that we shouldn't give user that control. They should be only allowed to show what the component allows them to do - that is aligned with Flamingo. If we allow them to inject whatever they want to - we're screwed :/

Reply:
I don't think it's gonna be a problem because, in the end, we will also need this flexibility

Reply:
It's like every API. It should be as exposed as needed. If we allow users to inject whatever they want - there's no point of this whole effort that Juan is doing.

Anyways, I've adjusted this component to accept the type of the value